cfme/storage/volume.py

```diff
@@ -362,6 +404,7 @@ def status(self):
         """
         view = navigate_to(self.parent, 'All')
         view.toolbar.view_selector.select("List View")
+        view.browser.refresh()
```

Comment:
Q: I think its not needed. can you please try it one time.

Reply:
I tried it and it was needed. When I call wait_for volume.status then it doesn't refresh page every time I call it.

Reply:
aaaaah understand need browser refresh we are on same page and try to check updated status. :+1: